README.md

```diff
@@ -11,6 +11,10 @@ presents the data collected by those processes.
 It also provides an interface on some markers of a watched AWS
 account's health from a security point of view.
 
+## Trusted Advisor
+Security HQ uses information from AWS Trusted Advisor.
+This might not be as up to date as one might wish and because noticeable for S3 buckets.
```

Comment:
Can we clarify what the update cadence is, whether it's possible to refresh this data and what the restrictions are around that?

Reply:
I put the comment more it to give further onlookers a hint, not because I yet know how it works.